syntax = "proto3";

service Informer {
  rpc GetPort (GetPortReq) returns (GetPortRep) {}
  rpc GetDescription (GetDescriptionReq) returns (GetDescriptionRep) {}
}

message GetPortReq {
  string service = 1;
}

message GetPortRep {
  int32 port = 1;
}

message GetDescriptionReq {
  string service = 1;
}

message GetDescriptionRep {
  string name = 1;
  string port_and_protocol = 2;
  string aliases = 3;
  string comments = 4;
}
